clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -1660,19 +1663,18 @@ protected void processCommonConsumerAction(
           partitionIngestionExceptionList.set(partition, null);
         } else {
           LOGGER.info(
-              "{} No need to reset offset by Kafka consumer, since the consumer is not "
-                  + "subscribing Topic: {} Partition Id: {}",
+              "{} No need to reset offset by Kafka consumer, since the consumer is not " + "subscribing: {}",
               consumerTaskId,
-              topic,
-              partition);
+              topicPartition);
         }
         kafkaDataIntegrityValidator.clearPartition(partition);
-        storageMetadataService.clearOffset(topic, partition);
+        storageMetadataService.clearOffset(topicPartition.getPubSubTopic().getName(), partition);
```

Comment:
I think it would be good to refactor all functions of the `OffsetManager` interface to use `PSTP`, but I think that should be left out of scope of the current PR given how big it already is... LMK if you prefer that I lump it into this PR, and I can do it.

Reply:
That is fine, and this can be done in a following PR.